scripts/run-component-tests.js

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env node
+
+/**
+ * Script that simplifies the workflow of running unit tests for a component
+ * using Bazel. Here are a few examples:
+ *
+ *   node ./scripts/run-component-tests button            | Runs Material button tests
+ *   node ./scripts/run-component-tests overlay           | Runs CDK overlay tests
+ *   node ./scripts/run-component-tests src/cdk/a11y      | Runs CDK a11y tests
+ *   node ./scripts/run-component-tests a11y overlay      | Runs CDK a11y and overlay tests
+ *
+ * Supported command line flags:
+ *
+ *   --local    | If specified, no browser will be launched.
```

Comment:
Can we make this the default? Then invert the flag to something like `--no-headless`

Reply:
Do you mean that we do not run any browser _by default_? I don't think we have a way to launch Chrome through Bazel without headless (it seems to depend on the `$DISPLAY` env variable).

Reply:
I want the command to not pop open a browser window but to also run the tests without having to manually open the page (but you can still open the debug page if you need to)

Reply:
I don't think this is possible. The `@bazel/karma` rules decide based on the platform environment whether the browser runs in headless or non-headless mode. For me on Windows, it always runs headless.